### PR TITLE
Omnichurch beacon update

### PR DESCRIPTION
**Pull request by DustScoundrel**
## About the PR
- Fixed the connection to the outer APC to ensure the entire station is powered.
- Added two signs to the docking entrance welcoming visitors to the Beacon outpost, to give a little flavor.
- Changed names and descriptions of the APCs and air alarms to help define rooms.

## Media
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**
N/A

### CODE REVIEW

File: Resources/Maps/Bluespace/grifty.yml
Comment (Cheackraze) [inline Resources/Maps/Bluespace/grifty.yml:1]: Lookers like you committed a few extra files

File: Resources/Maps/_NF/POI/beacon.yml [hunk 1]
```diff
@@ -8863,15 +9052,13 @@ entities:
...
-- proto: WarpPoint
+- proto: WarpPointShip
   entities:
   - uid: 1029
     components:
     - pos: 0.5,-0.5
       parent: 1
       type: Transform
-    - location: Omnichurch Beacon
-      type: WarpPoint
```
Comment (dvir001) [inline Resources/Maps/_NF/POI/beacon.yml:8874]: Warp broken again

File: Resources/Maps/_NF/POI/beacon.yml [hunk 2]
```diff
@@ -4311,8 +4485,6 @@ entities:
     - pos: -6.5,3.5
       parent: 1
       type: Transform
-    - name: Omnichurch Beacon
-      type: FaxMachine
```
Comment (dvir001) [inline Resources/Maps/_NF/POI/beacon.yml:4315]: Fax broken again.

**Review by dvir001 — CHANGES_REQUESTED** [review]
I moved the file to the new location, deleted the extra files, I need you to please fix the warp and the fax so we can get this update in,

Thanks.

**Comment by dvir001** [comment]
@DustScoundrel If you can update your PR to solve conflicts its good to go.

**Comment by dvir001** [comment]
Moved this to a new PR https://github.com/new-frontiers-14/frontier-station-14/pull/880
So I can keep it merged with master.

**Comment by dvir001** [comment]
Closed this for the active PR.

**Pull request closed without merging**